pilot/pkg/networking/core/v1alpha3/cluster.go

```diff
@@ -703,3 +745,27 @@ func buildDefaultTrafficPolicy(env *model.Environment, discoveryType v2.Cluster_
 		},
 	}
 }
+
+// TODO: merge with that in ep_filters.go
+// lbWeightNormalize set LoadBalancingWeight with a valid value.
+func lbWeightNormalize(endpoints []endpoint.LbEndpoint) []endpoint.LbEndpoint {
```

Comment:
Do you want to move this to networking/util and use this function in both places?

Reply:
Thanks for the advice.